lib/importer/factory/object_factory.rb

```diff
@@ -25,11 +25,16 @@ def run
         object
       end
 
+      ## FOR CONSIDERATION: handle a row (i.e. Work) with more than one file:
+      ## currently the file_set is replaced on update
       def update
         raise "Object doesn't exist" unless object
+        @attr = update_attributes
```

Comment:
Why have you chosen to use an instance variable here? Why not a local?

Reply:
It is called in `attach_file_to_work` (line 137).